dashboard/lib/pd/certificate_renderer.rb

```diff
@@ -41,34 +41,22 @@ def self.render_workshop_certificate(enrollment)
     end
 
     private_class_method def self.course_name(workshop)
-      if workshop.csf?
-        [
-          {
-            string: workshop.course_name,
-            y: -30,
-            pointsize: 70,
-            width: 1600,
-            height: 100,
-          },
-          {
-            string: workshop.friendly_subject,
-            y: 65,
-            pointsize: 60,
-            width: 1600,
-            height: 100,
-          }
-        ]
-      else
-        [
-          {
-            string: workshop.course_name,
-            y: -10,
```

Comment:
Will the removal of this part have an impact on any certificates that previously used a y of -10?

Reply:
My understanding could be incorrect, but I think it is ok because we always use the same certificate png, and the reason it was y of -10 for just the course name is we were centering the course name in the blank space, but for the new version we have two lines at y = -30 and y = 65 (course name and subject).

Reply:
Ah, this is a workshop-specific renderer, so should be fine.